build/Makefile.tcmalloc

```diff
@@ -3,6 +3,10 @@ ifeq ($(UNAME),FreeBSD)
 EXTRA_LDFLAGS="-lexecinfo"
 endif
 
+ifeq ($(UNAME),Linux)
+EXTRA_LDFLAGS="-lunwind"
```

Comment:
FreeBSD VM?

Actually, are you sure you really need this now?
I'm thinking that this previously worked (and continues to work on Travis etc.), and so I'm a bit surprised that you need to do anything more than add the `--disable-minidebuginfo` flag for the unwind build.